docs/modules/user-guide/nav.adoc

```diff
@@ -28,4 +28,4 @@
 ** xref:referring-to-a-parent-devfile-in-a-devfile.adoc[]
 
 * xref:api-reference.adoc[]
-* xref:devfile-samples.adoc[]
+* xref:devfile-resources.adoc[]
```

Comment:
I am not sure, what is happening here but i get an err when i build the docs:

```
$ antora antora-playbook.yml
error: Duplicate nav in 2.1.0@devfile: modules/user-guide/nav.adoc
  1: /Users/maysun/dev/devfile/docs/docs/modules/user-guide/nav.adoc
  2: docs/modules/user-guide/nav.adoc in https://github.com/devfile/docs (ref: v2.1.x)
Add the --stacktrace option to see the cause.
```

Reply:
@maysunfaisal not sure what's going on. I can build it fine. Is this what you see for devfile resources when you build this branch? 

<img width="1407" alt="Screen Shot 2021-09-28 at 12 49 18 PM" src="https://user-images.githubusercontent.com/70717303/135130852-5527349d-b140-4072-b7e4-df36c9a71ba5.png">